CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # Changelog
 
+## v0.2.0
+
+FEATURES
+* [\#358](https://github.com/bnb-chain/greenfield-storage-provider/pull/358) feat: sp services add pprof
+* [\#379](https://github.com/bnb-chain/greenfield-storage-provider/pull/379) feat:block syncer add read concurrency support
```

Comment:
Add empty space afrter `:`

Reply:
it is commit content, change log should be same with the origin